src/main.cc

```diff
@@ -175,10 +175,26 @@ int main(int argc, char **argv) {
   }
 
   if (debug) {
-    std::cerr << "loading " << gltf_path << std::endl;
+    std::cerr << "loading " << scene_path << std::endl;
   }
 
-  auto [meshes, mesh_instances] = LoadMeshesFromGLTF(gltf_path);
+  const std::string OBJ_EXTENSION = ".obj";
+  const std::string GLTF_EXTENSION = ".gltf";
```

Comment:
GLTF can also live in a `.glb` file. Also, can you switch to the const naming convention from the style guide?
```suggestion
  const std::string kOBJExtension = ".obj";
  const std::string kGLBExtension = ".gltf";
  const std::string kGLBExtension = ".glb";
```